certdepot/init.go

```diff
@@ -0,0 +1,96 @@
+package certdepot
+
+import (
+	"strings"
+	"time"
+
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	"github.com/square/certstrap/depot"
+	"github.com/square/certstrap/pkix"
+)
+
+type InitOptions struct {
+	// The depot which stores the id/cert.
+	Depot depot.Depot
```

Comment:
take depot out? 

does it makes sense to collapse these option structures to a "certificate options," and then implement each of the operations as a *function that takes a deopot and returns an error"